released/packages/coq-switch/coq-switch.1.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+version: "1.0.1"
+maintainer: "vzaliva@cmu.edu"
+homepage: "https://github.com/vzaliva/coq-switch"
+dev-repo: "https://github.com/vzaliva/coq-switch.git"
+bug-reports: "https://github.com/vzaliva/coq-switch/issues"
+authors: ["Vadim Zaliva <vzaliva@cmu.edu>"]
+license: "MIT"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Switch"]
+depends: [
+  "coq" {>= "8.9.1"}
+  "coq-metacoq-template" {>= "1.0~alpha+8.9"}
+]
+tags: [
+  "category:Miscellaneous/Coq Extensions"
+  "date:2019-10-09"
```

Comment:
Pro. Would you mind adding a `"logpath:Switch"` tag too to guide the `Require` command?